### DISCUSSION TITLE
Why enable steam:removable-media as default for snap?

### DISCUSSION THREAD

**Discussion by War3Evo** [canonical/steam-snap#115] [General]
Hi, I'm curious why steam:removable-media is default enabled?  Is it required to run steam in snap?

**Comment by kenvandine** [comment]
It's needed for users that want to keep their game library on a USB drive.

**Reply by War3Evo** [reply]
Ok.  I now understand.  That is exactly what I should have NOT disabled... LOL, because I do want steam to off put the library to a removable drive.
